vivisect/base.py

```diff
@@ -247,7 +247,8 @@ def _handleADDFUNCTION(self, einfo):
 
     def _handleDELFUNCTION(self, einfo):
         self.funcmeta.pop(einfo)
-        self.func_args.pop(einfo)
```

Comment:
actually, if you give pop() a default value argument, it ceases to fail when the key isn't there and saves us the extra dictionary hash tree traversal.  this one ( and the one above... and all others... for all time.. ) should just become pop(thing,None)